modules/agar/src/main/ts/ephox/agar/api/Mouse.ts

```diff
@@ -46,17 +55,24 @@ const cMouseMove = cMouseMoveWith({ });
  * @deprecated use cMouseOutWith({ }) instead */
 const cMouseOut = cMouseOutWith({ });
 
+const triggerOn = <T extends Element>(container: SugarElement<Node>, selector: string, action: (ele: SugarElement<T>) => void): SugarElement<T> => {
+  const ele = UiFinder.findIn(container, selector).getOrDie();
```

Comment:
getOrDie message?

Reply:
This is a result, so the message is already in the error and `getOrDie` doesn't take an error message.